### DISCUSSION TITLE
the answer in discussions is sometimes cut off

### DISCUSSION THREAD

**Discussion by 3m4r5** [community/community#18237] [Mobile] [answered]
[screenshot](https://user-images.githubusercontent.com/63432932/173016675-862b4381-8746-409d-88c7-270a3ad52202.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the report, I can't reproduce this on my end. What version of the app are you running, and what size device are you on?

**Reply by 3m4r5** [reply]
I'm using iPhone 8 Plus, Github 1.68.0

**Reply by 3m4r5** [reply]
[another example](https://user-images.githubusercontent.com/63432932/173165451-330d608f-6b34-4467-8ffd-69d37a0dcee3.png)

**Reply by brianlovin** [reply]
Thanks for the additional context. I'm passing this along to our eng team to see if we can figure out what might be going on here!
